netmiko/ssh_dispatcher.py

```diff
@@ -212,7 +209,6 @@
 CLASS_MAPPER["cisco_xr_telnet"] = CiscoXrTelnet
 CLASS_MAPPER["dell_dnos6_telnet"] = DellDNOS6Telnet
 CLASS_MAPPER["dell_powerconnect_telnet"] = DellPowerConnectTelnet
-CLASS_MAPPER["dlink_ds_telnet"] = DlinkDSTelnet
```

Comment:
shouldn't be removing this.

Reply:
Please check my new commit, should of fixed it